packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -107,14 +110,32 @@ function AnimalInventory({ isCompactSideMenu }: AnimalInventoryProps) {
         label: '',
         format: (d: AnimalInventory) => <Cell kind={CellKind.RIGHT_CHEVRON_LINK} path={d.path} />,
         columnProps: {
-          style: { width: '40px', padding: `0 ${isMobile ? 8 : 12}px` },
+          style: { width: '40px', padding: `0 ${isDesktop ? 12 : 8}px` },
         },
         sortable: false,
       },
     ],
-    [t, isMobile],
+    [t, isDesktop],
   );
 
+  const makeAnimalsSearchableString = (animal: AnimalInventory) => {
+    return [animal.identification, animal.type, animal.breed, ...animal.groups, animal.count]
+      .filter(Boolean)
+      .join(' ');
+  };
+
+  const [searchAndFilteredInventory, searchString, setSearchString] = useSearchFilter(
+    filteredInventory,
+    makeAnimalsSearchableString,
+  );
+
+  const searchProps: SearchProps = {
+    searchString,
+    setSearchString,
+    placeHolderText: t('ANIMAL.SEARCH_INVENTORY_PLACEHOLDER'),
+    searchResultsText: t('ANIMAL.SHOWING_RESULTS_WITH_COUNT', { count: filteredInventory?.length }),
```

Comment:
Since you renamed, `count` should be `searchAndFilteredInventory.length`

Reply:
fixed ! 🙏